docs/UserGuide.adoc

```diff
@@ -432,7 +432,7 @@ the bottom of the list.
 Example:
 
 * `activityAdd n/Sentosa Outing d/16/02/2019 1000 l/Harbour Front MRT de/Bring swim suits.`
-* `activityAdd n/HTML workshop d/1402019 l/ICube Auditorium ic/A0123456Z`
+* `activityAdd n/HTML workshop d/14/02/2019 l/ICube Auditorium ic/A0123456Z`
```

Comment:
the ic field no longer applicable right?

Reply:
ok i'll remove it